test/integration/kubernetes_deploy_test.rb

```diff
@@ -802,6 +802,23 @@ def test_ejson_secrets_respects_no_prune_flag
     ejson_cloud.assert_secret_present('ejson-keys', managed: false)
   end
 
+  def test_partials
+    result = deploy_dir_without_profiling(
```

Comment:
Our original discussion got lost, but I see why you can't use `deploy_fixtures` here. The best helper for this case would be `deploy_raw_fixtures("test-partials", bindings: { 'supports_partials' => 'true' })`. Incidentally, we should make a partial reference custom bindings (e.g. "supports_partials" in that command) to prove that works.

We also need another test about what happens when a referenced partial is not found (you can add at template to the 'invalid' fixture set for that purpose).